src/main/webapp/views/home.html

```diff
@@ -13,14 +14,43 @@
     <script src="../scripts/popper.min.js"></script>
     <script src="../scripts/bootstrap.bundle.min.js"></script>
 
+    <script src="../scripts/tutorial.js"></script>
     <script src="../scripts/auth.js"></script>
     <script src="../scripts/navbar.js"></script>
     <script src="../scripts/onload.js"></script>
   </head>
   <body>
-    <div id="home_container">
-      <div id="home_wrapper">
+    <div id="home_container" class="h-100">
+      <div id="home_wrapper" class="h-100">
         <div id="navbar_container"></div>
+        <br><br><br><br>
+        <div class="row align-items-center justify-content-center">
```

Comment:
An example of my previous comment:
```suggestion
        <div class="row align-items-center justify-content-center mt-6">
```